app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -203,4 +210,39 @@ private static void openChannelFragment(final Fragment fragment,
                 fragment.requireActivity().getSupportFragmentManager(),
                 item.getServiceId(), uploaderUrl, item.getUploaderName());
     }
+
+    /////////////////////////////////////////////
+    // helper functions                        //
+    /////////////////////////////////////////////
+
+    private static void fetchItemInfoIfSparse(final Fragment fragment,
```

Comment:
Why is the fragment used as param here? The only usage for fragment is `fragment.getContext()` in this method.